administrator/language/en-GB/com_users.ini

```
@@ -22,6 +22,10 @@ COM_USERS_BATCH_SET="Move To Group"
 COM_USERS_CATEGORIES_TITLE="User Notes: Categories"
 COM_USERS_CATEGORY_HEADING="Category"
 COM_USERS_CONFIGURATION="Users: Options"
+COM_USERS_CONFIG_ALLOWED_POSITIONS_BACKEND_DESC="When displaying the back-end Multi-factor Authentication page all modules will be hidden except those in the positions selected here. Please note that modules in the <code>title</code> position are always shown: this is required to show the back-end page icon and title."
+COM_USERS_CONFIG_ALLOWED_POSITIONS_BACKEND_LABEL="Allowed backend module positions"
+COM_USERS_CONFIG_ALLOWED_POSITIONS_FRONTEND_DESC="When displaying the front-end Multi-factor Authentication page all modules will be hidden except those in the positions selected here."
```

```suggestion
COM_USERS_CONFIG_ALLOWED_POSITIONS_FRONTEND_DESC="When displaying the frontend Multi-factor Authentication page all modules will be hidden except those in the positions selected here."
```

As per the style guide. "One word not hyphenated or two words."